packages/turbo-workspaces/README.md

```diff
@@ -0,0 +1,44 @@
+# `@turbo/workspace-convert`
+
+Easily convert your monorepo between package managers. Currently only supports monorepos using either npm, yarn, or pnpm workspaces.
+
+To get started, open a new shell and run:
+
+## CLI
+
+```sh
+npx @turbo/workspace-convert [flags...] [<dir>]
+```
+
+Then follow the prompts you see in your terminal.
+
+## Node API
+
+Methods are also available via the Node API:
+
+```js
+import { convertMonorepo, getWorkspaceDetails } from "@turbo/workspace-convert";
+
+// detect the package manager
+const project = getWorkspaceDetails({
+  root: process.cwd(),
+});
+
+// if the package manager is not pnpm, convert to pnpm
+if (project.packageManager !== "pnpm") {
+  await convertMonorepo({
+    root: process.cwd(),
+    to: "pnpm",
+    options: {
+      dry: false,
+      install: true,
+    },
+  });
+}
+```
+
+See [src/index.ts](src/index.ts) for details
```

Comment:
"read the source" isn't documentation. 😜

```suggestion
```

Reply:
😄 fair.